src/app/grid/grid-validator-service-extended/grid-validator-service-extended.component.html

```diff
@@ -0,0 +1,72 @@
+<div class="grid__wrapper">
+    <igx-grid #grid1 [data]="data" [width]="'100%'" [height]="'480px'" [autoGenerate]="false" [batchEditing]="true"
+        [primaryKey]="'id'" (onFormGroupCreate)='formCreateHandler($event)'>
+        <igx-column field="Avatar" header="Photo" dataType="string" width="80" [editable]="false">
+            <ng-template igxCell let-cell="cell">
+                <div class="cell__inner avatar-cell">
+                    <igx-avatar [src]="cell.row.data.avatar" [roundShape]="true" size="small"></igx-avatar>
+                </div>
+            </ng-template>
+        </igx-column>
+        <igx-column field="name" forbiddenName="josh" header="Name" [editable]="true" required>
```

Comment:
I don't think validating against a static name makes much sense in this data context.  I see no reason why a particular employee name would be forbidden. 

The other custom validators seem to be enogh to show how to do custom and cross field validation and they make more sense so this one can be removed IMO.